bwrap.xml

```diff
@@ -381,6 +385,12 @@
 	Write information in JSON format about the sandbox to FD.
       </para></listitem>
     </varlistentry>
+    <varlistentry>
+      <term><option>--json-status-fd <arg choice="plain">FD</arg></option></term>
+      <listitem><para>
+	Write information in JSON format about the sandbox to FD.
```

Comment:
That seems pretty vague, and is a duplicate of the `--info-fd` documentation.

From #257, #286, #293, #323 and the implementation, the idea seems to be:

* we write multiple JSON documents to *FD*, one per line (["JSON lines" format](https://jsonlines.org/))
* each line is a single JSON Object
* when we have started the child process inside the sandbox, we write an object with the `child-pid` to the `--json-status-fd` (this duplicates the older `--info-fd`)
* if available, the namespace IDs are also included in the object with the `child-pid` (again, this duplicates the older `--info-fd`)
* when the child process inside the sandbox exits, we write an object with the `exit-code`, which is the exit status of the child in the usual shell encoding (*n* if it exited normally with status *n*, 128+*n* if it was killed by signal *n*), and then close the `--json-status-fd`
* other members may be added to those objects in future, so readers must ignore members they do not understand
* other lines before the `exit-code` may be added in future, so readers must ignore lines they do not understand

Reply:
OT: indention in bwrap.xml has tabs and spaces mixed.